app/courses/page.tsx

```diff
@@ -101,4 +118,25 @@ const Path = ({
   );
 };
 
+const Banner = () => {
+  return (
+    <Flex
+      bg="blue.500"
+      color="white"
+      alignItems="center"
+      justifyContent="center"
+      p={4}
+    >
+      <Heading fontSize="lg" mr={4}>
+        Get unlimited access for free
+      </Heading>
+      <Link href={"/signup"}>
```

Comment:
Instead of linking to `/signup` we need to call the `signUp` function from `import { signUp } from 'next-auth/react'` like in main. One day.